abel/tools/center.py

```diff
@@ -276,10 +276,11 @@ def find_center_by_convolution(IM, **kwargs):
     len_conv = len(conv_0)/2
 
     #Take the first max, should there be several equal maxima.
-    center = (np.argmax(conv_0)/2, np.argmax(conv_1)/2)
+    # 10May16 - axes swapped - check this
```

Comment:
I think that what you are doing is correct, but confusing. Above, when you take `np.sum(IM, axis=0)`, you are taking a sum along axis 0, thereby "collapsing" that axis. So, the projection actually runs along axis 1. I would probably label QL_raw0 as QL_raw1 and vice versa in the two lines where they are created.